lib/tasks/petitions.rake

```diff
@@ -88,5 +88,11 @@ namespace :epets do
         end
       end
     end
+
+    desc "Email updated privacy policy"
+    task :email_privacy_policy_updates, [:date] => :environment do |_task, args|
+      date = args.date&.to_date || Date.new(2021, 3, 1)
```

Comment:
The `created_at` attribute is a timestamp so might be better represented as that and construct the argument like this:

``` ruby
time = (args.time || "2021-03-01").in_time_zone
EmailPrivacyPolicyUpdatesJob.perform_now(time: time)
```

Reply:
I like..amended!